.github/workflows/update.yml

```diff
@@ -0,0 +1,30 @@
+name: Update cve-feed-osv repo
+on:
+  schedule:
+  - cron: "0 */6 * * *"
+  workflow_dispatch:
+
+jobs:
+  update:
+    name: Update cve-feed-osv
+    runs-on: ubuntu-latest
+    env:
+      REPOSITORY_OWNER: ${{ github.repository_owner }}
```

Comment:
If I remember correctly, we don't need to pass `github.repository_owner` as an environment variable since it's already set. See https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables.

I'm okay if you want to explicitly set it for clarity,.